e2e/test/provisioning/ProvisioningCertificateValidationE2ETest.cs

```diff
@@ -19,108 +18,93 @@ public class ProvisioningCertificateValidationE2ETest : E2EMsTestBase
         [LoggedTestMethod]
         public async Task ProvisioningServiceClient_QueryInvalidServiceCertificateHttp_Fails()
         {
-            using (var provisioningServiceClient =
-                ProvisioningServiceClient.CreateFromConnectionString(
-                    Configuration.Provisioning.ConnectionStringInvalidServiceCertificate))
-            {
-                Query q = provisioningServiceClient.CreateEnrollmentGroupQuery(
-                    new QuerySpecification("SELECT * FROM enrollmentGroups"));
+            using var provisioningServiceClient = ProvisioningServiceClient.CreateFromConnectionString(
+                Configuration.Provisioning.ConnectionStringInvalidServiceCertificate);
+            Query q = provisioningServiceClient.CreateEnrollmentGroupQuery(
```

Comment:
I know this was already here but I think one letter variables are not the best choice. Rename q to query?

Reply:
I agree the variable name could be better. I'm not going too deep into this file clean up.